src/umpire/interface/umpire_shroud.yaml

```diff
@@ -130,6 +136,10 @@ declarations:
         options:
           F_create_generic: False
         cxx_template:
+        - instantiation: <umpire::strategy::DynamicPoolMap>
+          format:
+            template_suffix: _map_pool
```

Comment:
This should just be "_pool" to avoid breaking the C/Fortran API.

Reply:
whoopsies.. i thought i was being cool lol i'll change that